### PR TITLE
Develop to main Merge

### PR DESCRIPTION
**Pull request by govindsi**

### CODE REVIEW

File: common/scripts/mesh-11s.sh [hunk 1]
```diff
@@ -107,7 +108,7 @@ network={
     psk="$5"
     key_mgmt=SAE
     ieee80211w=2
-    mesh_fwding=0
+    #mesh_fwding=0
```
Comment (joenpera) [inline common/scripts/mesh-11s.sh:111]: Do we still have envs where this is not working?

Comment (govindsi) [reply common/scripts/mesh-11s.sh:111]: As supplicant change is now available in buildroot, we can remove this WAR.
I will raise the change in dev.

**Review by martin-tii — APPROVED** [review]

**Review by joenpera — CHANGES_REQUESTED** [review]
Waiting WAR removal

File: common/scripts/mesh-11s.sh (hunk 1 above)
Comment (govindsi) [reply common/scripts/mesh-11s.sh]: WAR revert Merged in dev.

**Review by joenpera — APPROVED** [review]

**Review by martin-tii — APPROVED** [review]

**Review by pemanty — APPROVED** [review]

**Pull request merged**